lib/rust/prelude/src/serde.rs

```diff
@@ -19,6 +19,18 @@ where
     serde_json::from_value(json_value).or_else(|_error| Ok(Ret::default()))
 }
 
+/// Deserialize a JSON value that is either of `Ret` type or equals `null`. A `null` is converted
+/// to a default value of `Ret` type.
+#[cfg(feature = "serde_json")]
+pub fn deserialize_null_as_default<'d, Ret, D>(d: D) -> Result<Ret, D::Error>
+where
+    for<'e> Ret: Default + Deserialize<'e>,
+    D: serde::Deserializer<'d>, {
+    let option_value = Option::deserialize(d)?;
+    Ok(option_value.unwrap_or_default())
+}
+
+
```

Comment:
If the field is missing, then I assume this behaves as it would be null?

Reply:
If and only if used together with `#[serde(default)]`.